### PR TITLE
chore(linux): Use dependency on core in ibus-keyman/build.sh

### PR DESCRIPTION
**Pull request by ermshiperete**
@keymanapp-test-bot skip

### CODE REVIEW

**Comment by ermshiperete** [comment]
@mcdurdin I would have expected https://github.com/keymanapp/keyman/pull/8423/files#diff-dc0124e5b672719c8d5807ab2f85b6b62db28e9bc17aeb0bfc6bbd61bfdbde3c to work now, but it's still skipping Core. Am I doing something wrong?

In `_builder_item_in_glob_array` the comparison `configure:* == `*:arch` fails when run in the script, but it succeeds when I try it on the command line :confused:

File: linux/ibus-keyman/build.sh [hunk 1]
```diff
@@ -18,11 +18,8 @@ builder_describe \
   "build" \
   "test" \
   "install                   install artifacts" \
-  "uninstall                 uninstall artifacts"
-
-# We can't yet depend on core until it moved to the new build.sh syntax
-# (currently it doesn't know some parameters that we're passing)
-#  "@/core configure build"
+  "uninstall                 uninstall artifacts" \
+  "@/core :arch"
```
Comment (mcdurdin) [inline linux/ibus-keyman/build.sh:22]:
```suggestion
  "@/core:arch"
```

Comment (ermshiperete) [reply linux/ibus-keyman/build.sh]: 🤦‍♂️ Of course. Thanks!

**Review by mcdurdin — APPROVED** [review]
LGTM

**Comment by keyman-server** [comment]
Changes in this pull request will be available for download in [Keyman version 17.0.68-alpha](https://keyman.com/downloads/releases/alpha/17.0.68)

**Pull request merged**